008-infinispan-getting-started/src/main/java/org/acme/quickstart/InfinispanClientApp.java

```diff
@@ -0,0 +1,31 @@
+package org.acme.quickstart;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+import io.quarkus.runtime.StartupEvent;
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.commons.configuration.XMLStringConfiguration;
+import org.slf4j.Logger;
```

Comment:
use import org.jboss.logging.Logger;

details in https://quarkus.io/guides/logging